extensions/amp-story/1.0/progress-bar.js

```diff
@@ -181,6 +188,14 @@ export class ProgressBar {
       true /** callToInitialize */
     );
 
+    this.storeService_.subscribe(
```

Comment:
Lets call `onIndexStateUpdate_` inside the existing `updateProgress` method instead of reacting to the store event:
https://github.com/ampproject/amphtml/blob/fbf807d68234e5e0ab465edfdfac24ee306ae38a/extensions/amp-story/1.0/progress-bar.js#L490

Since that method already listens for when there is a change in the current page (generally minimizing code paths is a good idea)

Reply:
Renamed `onIndexStateUpdate_` to `setAriaLabelByIndex_` and moved inside `updateProgress` method.

Reply:
Thanks for the feedback @Enriqe! I must be doing something wrong with my test 🙁 

```js
it('should have an aria-label attribute on the progress bar', () => {
    systemLayer.build();

    expect(
      systemLayer.querySelector('.i-amphtml-story-progress-bar')
    ).to.not.be.null;
  });
```
🔴 `TypeError: systemLayer.querySelector is not a function`

Calling `getShadowRoot`
```js
it('should have an aria-label attribute on the progress bar', () => {
    systemLayer.build();

    expect(
      systemLayer.getShadowRoot().querySelector('.i-amphtml-story-progress-bar')
    ).to.not.be.null;
  });
```
🔴 `AssertionError: expected null not to be null`

Selectors for other elements appear to get the element as expected

```js
it('should have an aria-label attribute on the progress bar', () => {
    systemLayer.build();

    expect(
      systemLayer
        .getShadowRoot()
        .querySelector('.i-amphtml-story-share-control')
    ).to.not.be.null;
  });
```
🟢 `pass`

Once I can get the progress bar `<ol>` element, I'll check that it assigns the expected `aria-label` and updates on page navigation.